### DISCUSSION TITLE
AStats cookie/php

### DISCUSSION THREAD

**Discussion by johnlx86** [lukasmoldon/astats-bot#2] [Q&A] [answered]
As already asked in issue https://github.com/lukasmoldon/astats-bot/issues/1, I also cant find the Astats cookie.

How can I find the required information using Firefox? Could you explain this step a bit more in detail?

**Comment by lukasmoldon** [comment, marked as answer]
First of all thank you for your feedback.

I added a detailed explanation to the readme in https://github.com/lukasmoldon/astats-bot/commit/0f2ad78087a26ae8b356b43caefd36861ca944c3. Does this solve your problem?

**Reply by johnlx86** [reply]
Thanks, it is now working properly!
